### DISCUSSION TITLE
Mark text in nushell

### DISCUSSION THREAD

**Discussion by jdfa** [nushell/nushell#7867] [General]
I am using Windows and used to mark words using Ctrl+Shift+ArrowLeft/ArrowRight. That behavior helps me during any editing workflow across system. Also that works in powershell and CommandPrompt. Usually I am combining that with copy/paste commands.
Is there any way I can achieve same functionality in nushell itself? I didn't find anything about that in issues or documentation.
Right now I see some workaround with using WindowsTerminal marking, but that will not help to cut text.

**Comment by fdncred** [comment]
This would be functionality in reedline and crossterm, which nushell uses. I do not think there is any selection by keyboard mechanism within nushell right now.

**Reply by jdfa** [reply]
Yes, looks like it is already in scope of reedline. Found this: https://github.com/nushell/reedline/issues/380

**Reply by fdncred** [reply]
ya, we'd love to have someone contribute this feature.
